src/hooks/auth/useAuth.ts

```diff
@@ -218,15 +221,20 @@ const useAuth = (): UseAuthReturn => {
 
   const logout = async (): Promise<void> => {
     await Promise.all([
-      AsyncStorage.removeItem(LocalStorageKey.AUTH),
+      AsyncStorage.clear(),
       removeKeys(),
       auth().signOut(),
       disconnect(),
+      resetPin(),
+      resetNewPin(),
     ])
 
     setCurrentUser(undefined)
     setRestoreLoading(false)
     setUser(undefined)
+
+    setLoading(true)
```

Comment:
why?

Reply:
After logout, go to the main screen, the user can interact with the menu. To prevent that momentary timing, I temporarily added loading